### PR TITLE
Update 0.7

### PR DESCRIPTION
**Pull request by mstonefield**
Adding a bunch of things

### CODE REVIEW

File: common/ai_budget/00_minerals_budget.txt
Comment (cemal-efe-g) [inline common/ai_budget/00_minerals_budget.txt:269]: This needs a full rework, with the army update

File: common/ai_budget/00_minerals_budget.txt
Comment (cemal-efe-g) [inline common/ai_budget/00_minerals_budget.txt:336]: I remember special defense armies, but that be something that will be implemented in the future.

File: common/ai_budget/00_minerals_budget.txt
Comment (cemal-efe-g) [inline common/ai_budget/00_minerals_budget.txt:358]: Special army

File: common/ai_budget/00_minerals_budget.txt
Comment (cemal-efe-g) [inline common/ai_budget/00_minerals_budget.txt:552]: Lithoids?

File: common/ai_budget/00_minerals_budget.txt
Comment (cemal-efe-g) [inline common/ai_budget/00_minerals_budget.txt:632]: Remnant code will never trigger

File: common/ai_budget/00_unity_budget.txt
Comment (cemal-efe-g) [inline common/ai_budget/00_unity_budget.txt:42]: Geth sphere might need to be added here, now that the AI is tech rushing. We probably want to wait until scripted triggers review though.

**Review by cemal-efe-g — CHANGES_REQUESTED** [review]
AI economy review

File: common/ai_budget/00_dark_matter_budget.txt
Comment (mstonefield) [reply common/ai_budget/00_dark_matter_budget.txt:1]: Since dark matter and some vanilla rare resources are still in the game I included them for simplicity's sake. But once we get our own custom resources fully put in I can easily make these files for those and remove the vanilla references

File: common/ai_budget/00_energy_budget.txt
Comment (mstonefield) [reply common/ai_budget/00_energy_budget.txt:14]: We actually do still have gaia worlds in and able to be made via terraforming and any of the abandoned terraforming project events so it was included.

File: common/ai_budget/00_energy_budget.txt
Comment (mstonefield) [reply common/ai_budget/00_energy_budget.txt:113]: We use ringworld districts currently. And the collector base has the ringworld designation

Comment (mstonefield) [reply common/ai_budget/00_energy_budget.txt:113]: Mostly because it enables them to actually have income with the limited resources available

File: common/ai_budget/00_energy_budget.txt
Comment (mstonefield) [reply common/ai_budget/00_energy_budget.txt:209]: Not sure, but included it with the file anyways tbh.

File: common/ai_budget/00_energy_budget.txt
Comment (mstonefield) [reply common/ai_budget/00_energy_budget.txt:239]: yeah terraforming is definitely going to get reworked methinks

File: common/ai_budget/00_energy_budget.txt
Comment (puckhead73) [reply common/ai_budget/00_energy_budget.txt:14]: I want to completely overhaul how terraforming works, and am currently debating the removal of Gaia worlds, I don't know that they really add anything, nor can I think of any worlds in ME lore that would classify as Gaia.

File: common/ai_budget/00_energy_budget.txt
Comment (puckhead73) [reply common/ai_budget/00_energy_budget.txt:239]: Oh hey look more discussion on terraforming. Yeah, I want to rip it out almost entirely, and have it pretty heavily nerfed, any terraforming in ME terms is really just throwing ice asteroids and stuff into planets to jumpstart the climate change or whatever, definitely nothing like what Stellaris presents.

File: common/buildings/00_capital_buildings.txt
Comment (cemal-efe-g) [inline common/buildings/00_capital_buildings.txt:43]: My only issue with buildings is that some of them don't seem to have the asari modifier, can we check that again, this is one of the examples @maxim-a-k @mstonefield

File: common/buildings/14_branch_office_buildings.txt
Comment (cemal-efe-g) [inline common/buildings/14_branch_office_buildings.txt:447]: I can't see pushbacks on the criminal syndicate buff we have done, did we revert that? I think that would be important for balance this patch.

File: common/buildings/btr_citadel_species_buildings.txt
Comment (cemal-efe-g) [inline common/buildings/btr_citadel_species_buildings.txt:119]: Was this buffed?

**Review by cemal-efe-g — CHANGES_REQUESTED** [review]
Building review

File: common/archaeological_site_types/btr_arc_sites.txt
Comment (puckhead73) [reply common/archaeological_site_types/btr_arc_sites.txt:27]: Yeah dig sites are something I'm going to be covering as part of my map review. My goal for it is to go over every doc that Jon threw together, as well as do an additional codex/wiki review for everything, and as I do I will be creating anomalies/dig sites/etc for each planet as needed. Also I mentioned this to Tex, but I can assign arch sites (including our starting ones) in the initializers, so we won't need an event to spawn them in at game start.

File: common/buildings/btr_citadel_species_buildings.txt
Comment (puckhead73) [inline common/buildings/btr_citadel_species_buildings.txt:164]: Need to remove Asari stuff from Turian Buildings

File: common/buildings/btr_citadel_species_buildings.txt
Comment (puckhead73) [inline common/buildings/btr_citadel_species_buildings.txt:367]: Are we supposed to have two sections for the Alliance?

File: common/buildings/btr_planetary_shields.txt
Comment (puckhead73) [inline common/buildings/btr_planetary_shields.txt:58]: Weight 0 means the AI will never build this, is this intended?

File: common/buildings/btr_planetary_shields.txt
Comment (puckhead73) [inline common/buildings/btr_planetary_shields.txt:268]: Should look at replacing this with EEZO.

File: common/component_templates/00_utilities_aux_distant_stars.txt
Comment (cemal-efe-g) [inline common/component_templates/00_utilities_aux_distant_stars.txt:20]: Did we do ship balance?

File: common/component_templates/00_utilities_reactors.txt
Comment (cemal-efe-g) [inline common/component_templates/00_utilities_reactors.txt:1141]: Was this accidentally added back or do we have dark matter?

File: common/component_templates/00_utilities_aux_distant_stars.txt
Comment (cemal-efe-g) [inline common/component_templates/00_utilities_aux_distant_stars.txt:20]: Did we do ship balance?

File: common/component_templates/00_utilities_shields.txt
Comment (cemal-efe-g) [inline common/component_templates/00_utilities_shields.txt:264]: Exotic gasses added in too, probably as part of the startech updates.

File: common/component_templates/00_utilities_shields.txt
Comment (cemal-efe-g) [inline common/component_templates/00_utilities_shields.txt:412]: Gasses

File: common/component_templates/00_utilities_shields.txt
Comment (cemal-efe-g) [inline common/component_templates/00_utilities_shields.txt:447]: Gases

File: common/component_templates/00_utilities_shields.txt
Comment (cemal-efe-g) [inline common/component_templates/00_utilities_shields.txt:607]: I'll not show every further resource you get the deal, every strategic just got added back in as part of the update.

File: common/component_templates/00_weapons_energy.txt
Comment (cemal-efe-g) [inline common/component_templates/00_weapons_energy.txt:267]: A similar problem again.

File: common/component_templates/00_weapons_extra_large.txt
Comment (cemal-efe-g) [inline common/component_templates/00_weapons_extra_large.txt:28]: Same stuff.

File: common/component_templates/00_weapons_missiles.txt
Comment (cemal-efe-g) [inline common/component_templates/00_weapons_missiles.txt:127]: Same.

File: common/council_treaties/fairxen_limitations_wip.txt
Comment (cemal-efe-g) [inline common/council_treaties/fairxen_limitations_wip.txt:5]: Let's remove everything related to farixen.

**Review by cemal-efe-g — CHANGES_REQUESTED** [review]
Component template reviews

File: common/component_templates/00_utilities_aux.txt
Comment (puckhead73) [inline common/component_templates/00_utilities_aux.txt:103]: Replacing with Helium 3 instead makes more sense

File: common/component_templates/00_utilities_aux.txt
Comment (puckhead73) [inline common/component_templates/00_utilities_aux.txt:108]: Replacing with Helium 3 instead makes more sense

File: common/component_templates/00_utilities_reactors.txt
Comment (puckhead73) [reply common/component_templates/00_utilities_reactors.txt:1141]: Dark matter immediately above this comment

File: common/council_treaties/fairxen_limitations_wip.txt
Comment (puckhead73) [reply common/council_treaties/fairxen_limitations_wip.txt:5]: Agreed. Also, file name has a typo :P

Comment (puckhead73) [reply common/council_treaties/fairxen_limitations_wip.txt:5]: And just to clarify, that's not saying we won't utilize this approach, just that since we're not getting into it currently, let's not include the files in the update

File: common/country_types/00_country_types.txt
Comment (puckhead73) [inline common/country_types/00_country_types.txt:578]: Obviously we're going to want to clean all of these up - probably is worth asking the question if we just want to comment all of the extraneous stuff like this out entirely so as to reduce the overhead the game compiles and loads. Only argument I can think of for retaining stuff like this would be for compatibility with event mods and things, which in my opinion is not the most compelling of arguments.

File: common/deposits/btr_citadel_species_planetary_deposits.txt
Comment (puckhead73) [inline common/deposits/btr_citadel_species_planetary_deposits.txt:231]: I'll be doing a full more in depth review of planetary deposits and their balance as I go through the map expansion and get everything else Jon had planned out implemented, but this all looks good.

File: common/deposits/btr_planetary_deposits.txt
Comment (puckhead73) [inline common/deposits/btr_planetary_deposits.txt:8195]: lol what happened here?

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:157]: motes

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:161]: motes

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:352]: crystals

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:356]: crystals

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:499]: gases

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:503]: gases

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:586]: gases

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:590]: gases

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:640]: crystals

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:644]: crystals

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:731]: crystals

File: common/districts/btr_districts.txt
Comment (puckhead73) [inline common/districts/btr_districts.txt:735]: crystals

File: common/economic_plans/03_post_science.txt
Comment (puckhead73) [inline common/economic_plans/03_post_science.txt:14]: We'll want to revisit these as well, if I'm understanding how this works properly, this might not work as intended considering we've removed exotic gases, motes, and crystals.

File: common/edicts/00_edicts.txt
Comment (puckhead73) [inline common/edicts/00_edicts.txt:14]: If we're going the Doomsday route for the Rakhana (I'm fairly certain that was the decision) we should repurpose this to use their origin.

File: common/edicts/01_campaigns.txt
Comment (puckhead73) [inline common/edicts/01_campaigns.txt:289]: Can probably comment these out for now pending the completion of the new resources full implementation.

**Review by maxim-a-k — APPROVED** [review]
Barring any unused resource changes, following an explanation from Tex I am confident the overhaul is good.

**Review by cemal-efe-g — CHANGES_REQUESTED** [review]
Descriptor Artefact

File: common/ai_budget/00_alloys_budget.txt
Comment (cemal-efe-g) [reply common/ai_budget/00_alloys_budget.txt:654]: @mstonefield was this resolved?

**Comment by cemal-efe-g** [comment]
I LOVE BTR!!!

**Review by cemal-efe-g — APPROVED** [review]
I LOVE BTR, LET'S GOO! Jk aside, great push everyone, in my opinion we are ready for the update.

File: common/static_modifiers/mec_asari_static_modifiers.txt
Comment (puckhead73) [inline common/static_modifiers/mec_asari_static_modifiers.txt:11]: Let's just merge this in with the BtR modifiers and get rid of this file

**Review by puckhead73 — APPROVED** [review]

**Comment by puckhead73** [comment]
Looks great! I found one MEC file we could eliminate, but as soon as we merge that into our own modifier file, I'll publish this sucker!

File: common/static_modifiers/mec_asari_static_modifiers.txt
Comment (mstonefield) [reply common/static_modifiers/mec_asari_static_modifiers.txt:11]: rgr

**Pull request merged**
